src/commands/test/test_cases.py

```diff
@@ -45,7 +45,7 @@ def __str__(self) -> str:
 
 
 @dataclass(frozen=True)
-class BrokenTest(CaseResult):
+class BrokenTestFile(TestCaseResult):
```

Comment:
much better name

Reply:
maybe `Suite`? I find it kinda problematic we use both names, we should settle for one of them